messaging-experiments.json

```diff
@@ -1116,5 +1116,279 @@
         }
       ]
     }
+  },
+  {
+    "id": "MULTI-STAGE-ABOUTWELCOME-V2-79-RELEASE",
+    "enabled": true,
+    "filter_expression": "env.version >= '79.' && (env.channel == 'default' || env.channel == 'beta')",
+    "targeting": "localeLanguageCode == \"en\" && [userId, \"message-multi-stage-aboutwelcome-v2-79\"]|bucketSample(0, 2000, 10000) && ((isFirstStartup && !('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)) || 'message-multi-stage-aboutwelcome-v2-79' in activeExperiments)",
+    "arguments": {
+      "slug": "message-multi-stage-aboutwelcome-v2-79",
```

Comment:
name space and slug should be updated with values from experimenter after signoffs in experimenter